sdk/core/core/src/az_http_response.c

```diff
@@ -268,3 +268,36 @@ void _az_http_response_reset(az_http_response* http_response)
   az_result result = az_http_response_init(http_response, http_response->_internal.http_response);
   (void)result;
 }
+
+// internal function to get az_http_response remainder
+static az_span _az_http_response_get_remaining(az_http_response const* response)
+{
+  return az_span_slice_to_end(response->_internal.http_response, response->_internal.written);
+}
+
+AZ_NODISCARD az_result az_http_response_write_span(az_http_response* response, az_span write)
```

Comment:
header/implementation parameter mismatch
```suggestion
AZ_NODISCARD az_result az_http_response_write_span(az_http_response* response, az_span source)
```